learn/guides/visual-studio-code-extension/debugging.md

```diff
@@ -156,7 +230,7 @@ Visual Studio Code allows you to debug Ballerina programs through the Ballerina
 - Strands
 - Expression Evaluation
 
->**Info** The Ballerina debugger is an open-source project and contributors are most welcome to get engaged with it via the [ballerina-lang](https://github.com/ballerina-platform/ballerina-lang) GitHub repository. If you encounter any difficulties when using this feature, feel free to create an issue on it.
+>**Info:** For more information on debugging features using VS Code, go to the [VS Code Documentation](https://code.visualstudio.com/docs/editor/debugging).
```

Comment:
Can we move this info section to the end of the `Using the Debugging Features` section?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/3385/commits/94c36033750840488ec1bd7dc58f95dcbb463de3